docs/source/markdown/podman-artifact-extract.1.md

```diff
@@ -0,0 +1,80 @@
+% podman-artifact-extract 1
+
+
+## WARNING: Experimental command
+*This command is considered experimental and still in development. Inputs, options, and outputs are all
+subject to change.*
+
+## NAME
+podman\-artifact\-extract - Extract an OCI artifact to a local path
+
+## SYNOPSIS
+**podman artifact extract** *artifact* *target*
+
+## DESCRIPTION
+
+Extract the blobs of an OCI artifact to a local file or directory. If the target
+path is a file or does not exists the artifact must either consists of one blob
+or if it has multiple blobs (layers) then the **--digest** or **--title** option
+must be used to select only a single blob. If the file already exists it will be
+overwritten.
+
+If the target is an directory (it must exist) all blobs will be copied to the
+target directory. As target file name the value from `org.opencontainers.image.title`
```

Comment:
With `--digest`, the file name probably should be always digest-based, so that the user knows where to find the output.

(Same with `--title`, but that happens ~automatically.)

Reply:
That makes the code actually simpler, I had the in my first version but then thought it would not be consistent. But yeah I guess you are right it makes sense to stick with the digest for users so they can predict the target name.